request_handlers.cc

```diff
@@ -298,49 +301,49 @@ RequestHandler::Status ReverseProxyRequestHandler::handleRequest(const Request&
     }
     else {
       redirect.AddRequestLine("GET", proxy_path, "HTTP/1.1");
-    } 
+    }
     redirect.AddHeader("Host", redirect_host);
     redirect.AddHeader("Accept", "*/*");
-    redirect.AddHeader("Connection", "close") ; 
+    redirect.AddHeader("Connection", "close") ;
```

Comment:
Looks like theres an extra space between the end of the function call and the semicolon here.